source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/new/display.tsx

```diff
@@ -121,32 +126,38 @@ export default class Display extends React.Component<IProps, IState> {
                     onChange={mut('language')} />
                 </Form.Field>
 
-                <Form.Field className='flex-50 m-l-md'>
+                <Form.Field className='flex-50 m-l-md m-b-md'>
                   <label>{t('project.type')}</label>
-                  <input required
-                    data-test-type
-                    value={type || ''}
-                    onChange={mut('type')} />
+                  <ApplicationTypeSelect
+                    selected={typeId}
+                    onChange={mut('typeId')}
+                  />
                 </Form.Field>
               </div>
 
-              <Form.Field className='m-t-xl'>
-                <div className='flex-row'>
-                  <div className='toggle-selector flex-row flex-50-sm flex-25-lg  justify-content-space-between'>
-                    <span className='bold'>{t('project.visibilityLabel')}</span>
-                    <Checkbox
-                      toggle
-                      data-test-visibility
-                      checked={isPublic}
-                      onChange={toggle('isPublic')}
-                      />
+              <div className='flex justify-content-space-between'>
+                <Form.Field className='flex-50 m-r-md'>
+                  <label>{t('project.projectDescription')}</label>
+                  <textarea value={description || ''} onChange={mut('description')}/>
+                </Form.Field>
+                <Form.Field className='flex-50 m-l-md m-t-md p-t-sm'>
+                  <div className='flex-row'>
+                    <div className='toggle-selector flex-row flex-grow justify-content-space-between'>
+                      <span className='bold'>{t('project.visibilityLabel')}</span>
+                      <Checkbox
+                        toggle
```

Comment:
are there types of checkboxes that aren't toggles? lol

Reply:
not in this application :D